### PR TITLE
Fix Clippy Warnings

### PR DESCRIPTION
**Pull request by JulianKnodt**

### CODE REVIEW

File: src/base/storage.rs [hunk 1]
```diff
@@ -41,6 +43,7 @@ pub type CStride<T, R, C = U1> =
 /// should **not** allow the user to modify the size of the underlying buffer with safe methods
 /// (for example the `VecStorage::data_mut` method is unsafe because the user could change the
 /// vector's size so that it no longer contains enough elements: this will lead to UB.
+///
```
Comment (Ralith) [inline src/base/storage.rs:46]: Stray empty line?

File: src/geometry/rotation.rs [hunk 2]
```diff
@@ -185,6 +185,11 @@ impl<T: Scalar, const D: usize> Rotation<T, D> {
     }
 
     /// A mutable reference to the underlying matrix representation of this rotation.
+    ///
+    /// # Safety
+    ///
+    /// Invariants of the rotation matrix should not be violated.
+    ///
```
Comment (Ralith) [inline src/geometry/rotation.rs:192]: Stray empty line?

**Review by Ralith — APPROVED** [review]
Thanks, this LGTM overall. Could you rebase?

File: nalgebra-sparse/src/ops/impl_std_ops.rs [hunk 3]
```diff
@@ -59,7 +59,7 @@ macro_rules! impl_sp_plus_minus {
             let mut result = $matrix_type::try_from_pattern_and_values(pattern, values)
                 .unwrap();
             $spadd_fn(T::zero(), &mut result, T::one(), Op::NoOp(&a)).unwrap();
-            $spadd_fn(T::one(), &mut result, $factor * T::one(), Op::NoOp(&b)).unwrap();
+            $spadd_fn(T::one(), &mut result, $factor, Op::NoOp(&b)).unwrap();
```
[the remark this answers is not in the document]
Comment (Ralith) [reply nalgebra-sparse/src/ops/impl_std_ops.rs:62]: Same. IMO if it compiles and the tests pass, then breakage is on us.

**Review by Ralith — APPROVED** [review]

File: nalgebra-sparse/src/ops/impl_std_ops.rs (hunk 3 above)
[the remark this answers is not in the document]
Comment (Andlon) [reply nalgebra-sparse/src/ops/impl_std_ops.rs:62]: I probably wrote this, and I have absolutely no idea. Probably started out as something else and gradually got rewritten into its current form, and then some old quirks remained. If all tests still pass, then I'd say go ahead :-)

**Pull request merged**